client/ayon_core/pipeline/create/context.py

```diff
@@ -2126,6 +2129,9 @@ def create(
             "productType": creator.product_type,
             "variant": variant
         }
+        if active is not None:
+            instance_data["active"] = bool(active)
```

Comment:
Why is needed specific cast to `bool`?

Reply:
Good question. Without it, it gave an issue in Houdini due to the fact that the HoudinI Placeholder node for the workfile template builder would return 1 or 0 instead of a boolean value - due to that being Houdini's return type for whatever reason for a Toggle parameter.

This meant, when creating the instance using the workfile template builder that the active `parm` would be "imprinted" on creation as an integer field instead of a boolean field.

I'd say this is also somewhat a separate bug or something we look into on how we're reading these toggle parms from Houdini to maybe ensure we're getting them as booleans instead. But since this was the safest way, and at least ensures the value is a boolean across all hosts (which it should be) I felt it was the easiest and best solution moving forward.

So basically:
- The Houdini placeholder node would have an `active` toggle
- It returns the value as `0` or `1`
- Which means `placeholder.data["active"]` is `0` or `1`
- Which means suddenly we're passing integer here instead of bool.
- Which influences how the value is imprinted on the created instance due to it being based on the type of the value in the instance data.

Hope that clarifies it.

@MustafaJafar - we might want to track this as a separate (non-urgent) issue for ayon-houdini.

Reply:
Well but that is related to houdini, I would say that in core it should expect `bool`, and it's job of houdini to convert it.

Reply:
I mean - it's similarly hard to just pass on `"0"` then and debug that. We could also have it explicitly error if it's not a boolean so we at least spot the mistakes directly and solve them. It does mean we'd need additional fixes on the Houdini side of things, which we should do anyway.

Shall I just to a type check if it is not None? Something like:
```python
if active is not None and not isinstance(active, bool):
    raise TypeError(f"Active value must be boolean, got: {active} ({type(active)})")
```

Reply:
After discussion in DM on discord we came to the conclusion to do the automatic conversion, but log a warning if the type is not a bool yet.

Reply:
Added warning: https://github.com/ynput/ayon-core/pull/774/commits/05f6b0f2ce000910004c3ce9913e17ec528bc04b